infra/config.go

```diff
@@ -1,26 +1,26 @@
 package infra
 
 import (
-	"encoding/json"
 	"io/ioutil"
 
 	"github.com/gogo/protobuf/proto"
 	"github.com/hyperledger/fabric-protos-go/msp"
+	"gopkg.in/yaml.v2"
 )
 
 type Config struct {
-	PeerAddrs     []string `json:"peer_addrs"`
-	OrdererAddr   string   `json:"orderer_addr"`
-	Channel       string   `json:"channel"`
-	Chaincode     string   `json:"chaincode"`
-	Version       string   `json:"version"`
-	Args          []string `json:"args"`
-	MSPID         string   `json:"mspid"`
-	PrivateKey    string   `json:"private_key"`
-	SignCert      string   `json:"sign_cert"`
-	TLSCACerts    []string `json:"tls_ca_certs"`
-	NumOfConn     int      `json:"num_of_conn"`
-	ClientPerConn int      `json:"client_per_conn"`
+	PeerAddrs     []string `yaml:"peer_addrs"`
```

Comment:
so if we need a map?
map[string]Node ...? same as current json?

Reply:
sure, although this will be done as part of your PR, since this one is only addressing config file format change